project.clj

```diff
@@ -119,6 +119,9 @@
   ;; code that we have.
   :classifiers  [["test" :testutils]]
 
+  ;; Inovke "lein test :all" to run all tests
+  :test-selectors {:default (fn [m] (not (:slow m)))}
```

Comment:
Oh, right - yes, we'll have to decide how often we want :slow to run -- though I suppose after this morning's discussion there might be some question about whether the test is as interesting now, aside from the fact that it makes it easy to perform some simple command memory processing dumps.